packages/model/addon/-private/model.js

```diff
@@ -472,26 +512,16 @@ const Model = EmberObject.extend(DeprecatedEvented, {
     @private
     @type {Object}
   */
-  currentState: RootState.empty, // defined here to avoid triggering setUnknownProperty
 
   /**
    @property _internalModel
    @private
    @type {Object}
    */
```

Comment:
We still have the comment, but do we no longer have the prop or do we just not initiate it here? Also for store

Reply:
we don't initialize here but still have it